recipe/meta.yaml

```diff
@@ -1,25 +1,32 @@
 {% set name = "gptcache" %}
-{% set version = "0.1.19" %}
+{% set version = "0.1.20" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/gptcache-{{ version }}.tar.gz
-  sha256: 6d7cbc4105354575385fec0efc18a6e5d4262e290c6b7b0d000ff741a82799de
+  sha256: 4e2f858b0291573d7dd04007754936a7e5a7c80fae318d30f47f2de7791244d8
+  patches:
+    - patches/0001-read-file-as-utf8.patch   # [win]
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
+  # Minimum Python requirement is 3.8, and openai isn't available on s390x.
+  skip: True # [py<38 or s390x]
 
 requirements:
+  build:
+    - m2-patch         # [win]
   host:
-    - python >=3.8
+    - python
     - pip
+    - wheel
+    - setuptools 66.0
   run:
-    - python >=3.8
+    - python
     - openai
     - numpy
     - cachetools
```

Comment:
Does not it also needs requests?
https://github.com/zilliztech/GPTCache/blob/0.1.20/requirements.txt#L4

Reply:
Yeah, it's definitely required https://github.com/search?q=repo%3Azilliztech%2FGPTCache%20requests&type=code